stdlib/2and3/pyexpat/__init__.pyi

```diff
@@ -0,0 +1,75 @@
+from typing import List, Tuple, Optional, Callable, Any, Protocol, Union, Dict
+
+import pyexpat.errors as errors
+import pyexpat.model as model
+
+EXPAT_VERSION: str  # undocumented
+version_info: Tuple[int, int, int]  # undocumented
+native_encoding: str  # undocumented
+features: List[Tuple[str, int]]  # undocumented
+
+class ExpatError(Exception):
+    code: int = ...
+    lineno: int = ...
+    offset: int = ...
+
+error = ExpatError
+
+class _Reader(Protocol):
+    def read(self, length: int) -> bytes: ...
+
+XML_PARAM_ENTITY_PARSING_NEVER: int
+XML_PARAM_ENTITY_PARSING_UNLESS_STANDALONE: int
+XML_PARAM_ENTITY_PARSING_ALWAYS: int
+
+_Model = Tuple[int, int, Optional[str], tuple]
+
+class XMLParserType:
```

Comment:
Should inherit from `object` to indicate that this is a new-style class in Python 2.